test/truefi2/TrueFiPool2.test.ts

```diff
@@ -967,99 +965,4 @@ describe('TrueFiPool2', () => {
       await expect(() => tusdPool.liquidExit(totalSupply.div(2))).to.changeTokenBalance(tusd, owner, totalValue.div(2).mul(penalty).div(10000))
     })
   })
-
-  describe('utilization', () => {
-    const includeFee = (amount: BigNumber) => amount.mul(10000).div(9975)
-
-    beforeEach(async () => {
-      await tusd.approve(tusdPool.address, includeFee(parseEth(1e5)))
-    })
-
-    it('utilization is 0%', async () => {
-      const depositedAmount = parseEth(1e5)
-      await tusdPool.join(depositedAmount)
-      expect(await tusdPool.utilization()).to.eq(0)
-    })
-
-    it('utilization is 100%', async () => {
-      await tusdPool.join(parseEth(1e5))
-      const loanForWholePool = await createApprovedLoan(
-        rater, tru,
-        stkTru, loanFactory,
-        borrower, tusdPool,
-        parseEth(1e5), DAY,
-        100, owner,
-        provider,
-      )
-      await lender.connect(borrower).fund(loanForWholePool.address)
-      expect(await tusdPool.utilization()).to.eq(100_00)
-    })
-
-    it('utilization is 50 %', async () => {
-      await tusdPool.join(parseEth(1e5))
-      const loanForPartOfPool = await createApprovedLoan(
-        rater, tru,
-        stkTru, loanFactory,
-        borrower, tusdPool,
-        parseEth(1e5).div(2), DAY,
-        100, owner,
-        provider,
-      )
-      await lender.connect(borrower).fund(loanForPartOfPool.address)
-      expect(await tusdPool.utilization()).to.eq(50_00)
-    })
-
-    describe('liquidRatio', () => {
-      beforeEach(async () => {
-        await tusd.approve(tusdPool.address, includeFee(parseEth(1e5)))
-        await tusdPool.join(parseEth(1e5))
-        const loanForPartOfPool = await createApprovedLoan(
-          rater, tru,
-          stkTru, loanFactory,
-          borrower, tusdPool,
-          parseEth(1e5).div(4), DAY,
-          100, owner,
-          provider,
-        )
-        await lender.connect(borrower).fund(loanForPartOfPool.address)
-      })
-
-      it('returns 0 if poolValue is 0', async () => {
-        expect(await usdcPool.liquidRatio()).to.eq(0)
-      })
-
-      it('equals 1 - utilization', async () => {
-        const expected = 100_00 - (await tusdPool.utilization()).toNumber()
-        expect(await tusdPool.liquidRatio()).to.eq(expected)
-      })
-    })
-
-    describe('proFormaLiquidRatio', () => {
```

Comment:
Let's keep unit tests for proFormaLiquidRatio. The current tests are split between utilization, liquidRatio, and proFormaLiquidRatio, but they should be rewritten + deduped under proFormaLiquidRatio. A set of tests that I think it's good to have:
- poolValue == 0 -> pFLR == 0 (no division by zero error)
- liquidValue == 0 -> pFLR == 0
- liquidValue == 40% * poolValue -> pFLR == 40% (don't use 50% because utilization would also pass this test)
- liquidValue == poolValue -> pFLR == 100%
- manually calculate `poolUtilization = (poolValue - liquidValue) * 100% / poolValue` -> `pFLR == 100% - poolUtilization)` (don't use 50%)
- pFLR(amount) before a loan == pFLR(0) after the loan

If you think of any other good ones feel free to add them.

Reply:
I brought back the tests that would work with the current setup (2 out of 3). When it comes to adding new ones, I think it can be done in a separate PR. This one is purely about removing unused code 😁 
I'll add a ticket to the board.